pkgs/top-level/all-packages.nix

```diff
@@ -17011,6 +17011,8 @@ in
 
   openrct2 = callPackage ../games/openrct2 { };
 
+  optparse-bash = callPackage ../development/libraries/optparse-bash { };
```

Comment:
This seem more of a tool than a library. We need a better path here...
--
Sorry for delay in answering. I understand this might be frustrating.

Reply:
Also, reviewers may be into nitpicking (sometimes to annoying degrees) to teach the project standards / best pratices. The goal is beyond the contribution. Consider it training. It eventually gets better.

Reply:
> This seem more of a tool than a library. We need a better path here...

Err.. You might be right. It just seemed strange at first.
https://nixos.org/manual/nixpkgs/unstable/#sec-organisation